detekt.yml

```diff
@@ -690,19 +688,6 @@ datadog:
       # region Gson
       - "com.google.gson.JsonParser.parseString(kotlin.String):com.google.gson.JsonParseException"
       # endregion
-      # region Kotlin Invoke
-      - "io.opentracing.Span.invoke():java.lang.Exception"
-      - "kotlin.Function0.invoke():java.lang.Exception"
-      - "kotlin.Function1.invoke():java.lang.Exception"
```

Comment:
strange that it has such signature, because `Function1` should have one argument, but it is zero here.